docker/tester-exec.sh

```diff
@@ -27,14 +29,10 @@ test_qemu() {
   PORT2=$((15000+unique_port))
   QEMU_MAC=52:54:00:${unique_mac:0:2}:${unique_mac:2:2}:${unique_mac:4:2}
 
-  qemu-system-x86_64 -device virtio-net-pci,netdev=net0,mac=$QEMU_MAC \
-  -netdev user,id=net0,hostfwd=tcp::$PORT1-:22,hostfwd=tcp::$PORT2-:23 \
-  -drive if=virtio,file=$FILENAME,format=raw -no-reboot -vga vmware \
-  -device virtio-rng-pci -drive if=pflash,format=qcow2,file=ovmf.qcow2 \
-  -vga std -cpu kvm64 -enable-kvm -m 256 -serial null --nographic &
+  nohup ./run-qemu.sh $PORT1 $PORT2 $QEMU_MAC $FILENAME &>run_qemu.log &
 
   QEMU_PID=$!
-  sleep 10 # Wait some time for QEMU to boot
+  sleep 20 # Wait some time for QEMU to boot and add an ssh key to image
```

Comment:
@mythi The connection seems to get refused if ssh connection is tried right away and if there is a small sleep before ssh, there will be password prompt. So I'll need to make a while loop if that is really needed. But first I'll try to make this stable. What I have tested that sleep doesn't affect it.